code/datums/achievements/_awards.dm

```diff
@@ -177,20 +176,45 @@
 	desc = "Don't worry, metagaming is all that matters."
 	icon = "elephant" //Obey the reference
 	database_id = ACHIEVEMENTS_SCORE
-	load_priority = AWARD_PRIORITY_LAST //See below
```

Comment:
`load_priority` has no reason to be now, since this was the only award with a different var value. Could you remove it in its entirety, defines included?